CommonTools/TrackerMap/interface/TrackerMap.h

```diff
@@ -196,29 +195,28 @@ class TrackerMap {
   bool enableLVProcessing;
   bool enableHVProcessing;
   bool tkMapLog;
-  int ndet;   //number of detectors
-  int npart;  //number of detectors parts
   std::string title;
   std::string jsfilename, infilename;
   std::string jsPath;
   bool psetAvailable;
+
   double phival(double x, double y) {
     double phi;
-    double phi1 = atan(y / x);
-    phi = phi1;
-    if (y < 0. && x > 0)
-      phi = phi1 + 2. * M_PI;
-    if (x < 0.)
-      phi = phi1 + M_PI;
     if (fabs(y) < 0.000001 && x > 0)
       phi = 0;
-    if (fabs(y) < 0.000001 && x < 0)
+    else if (fabs(y) < 0.000001 && x < 0)
       phi = M_PI;
-    if (fabs(x) < 0.000001 && y > 0)
+    else if (fabs(x) < 0.000001 && y > 0)
       phi = M_PI / 2.;
-    if (fabs(x) < 0.000001 && y < 0)
+    else if (fabs(x) < 0.000001 && y < 0)
       phi = 3. * M_PI / 2.;
-
+    else {
```

Comment:
Since this is the most common case wouldn't it make sense to test it first?
i.e., 
```
if (fabs(x) > 0.000001 & fabs(y) > 0.000001) 
{
 < evaluate tan >
}
else if{
  < test other cases > 
}
```

Reply:
Indeed. Thank you @mandrenguyen for the suggestion